wagtail/admin/static_src/wagtailadmin/scss/userbar.scss

```
@@ -111,6 +111,21 @@ $positions: (
     margin: 0;
   }
 
+  .w-userbar-axe-count {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $color-button-no;
+    border-radius: 100px;
+    color: $color-white;
+    font-size: 0.8125rem;
```

For new code, we use a set typographic scale to make sure text is large enough and usage of different font sizes is consistent.

```suggestion
    font-size: theme('fontSize.14');
```